src/test/kotlin/fi/metatavu/timebank/api/test/functional/tests/PersonsTest.kt

```diff
@@ -153,7 +153,9 @@ class PersonsTest: AbstractTest() {
         createTestBuilder().use { testBuilder ->
             val personTotalTimes = testBuilder.manager.persons.getPersonTotal(
                 personId = 2,
-                timespan = null
+                timespan = null,
+                before = LocalDate.now().minusDays(1).toString(),
```

Comment:
Does this set the time to an appropriate value e.g. will all timeEntries from the previous day be included?